GitVersionConfig.yaml

```diff
@@ -1,3 +0,0 @@
-mode: ContinuousDeployment
-assembly-versioning-scheme: MajorMinorPatch 
-#next-version: 1.0.0
```

Comment:
I'm saying I won't do anything before the PR is otherwise approved (since if it's not accepted, I'll maintain my fork and keep using NB.GV). If it's accepted, I may update the PR to use GitVersion if I have time, but to me it's a low priority because with this PR I'm already improving your versioning story (since you don't have one here). If you folks want it to be further improved (in your estimation) you are welcome to build on it if I don't have time to redo an improvement I have already made.